template.tex

```diff
@@ -1,4 +1,4 @@
-%% start of file `template.tex'.
+%% start of file `template.tex' as shipped with 2021/01/21 v2.1.0 modern curriculum vitae and letter document class
```

Comment:
Can you change the date please to the [ISO8601](https://de.wikipedia.org/wiki/ISO_8601) format? On Unix `date -u -I`.

Reply:
I can do that but it requires rewriting parts of the script in the make file introduced in the next pull request. 
The change in date format needs to be done to ALL files, not just the template file. I'd rather not change it manually here. 

The problem is that these pull requests are now sequential in time, so some things may change in the next pull request
The last pull request being the most up to date.

If you want to have a look at the user guide, use have a look at the document in the last pull request. 
The document will change a lot during the pull requests and it is pointless to check each step as each step but the last PR7 are obsolete. 
The last version still has red todos so that people can see what needs to be done, still.

Reply:
Okay maybe to help you out, you can do something what is called a "git rebase": https://git-scm.com/docs/git-rebase

Essentially what you do is you checkout the branch where you have all changes, create a new branch with `git branch update-userguide && git checkout update-userguide`. Then, when you are on the new branch, you do `git rebase -i master` (make sure your local master branch is up-to-date). You will get a text file that has all the commits you made. Now only selected the commits where you update the manual, for the other commits replace the `pick` with `drop`. Save the file, and you will have a clean branch.

If your commits do several things at once that you didn't want, you can bisect the commits and split them: https://git-scm.com/docs/git-bisect

Alternatively, what I usually do, I rebase, edit the commit I want to split (`edit` instead of `pick`), and run `git reset HEAD^`. Then, your changes stay, but the commit will be removed. Then you can commit each change individually.

Tools like gitg (https://wiki.gnome.org/Apps/Gitg/) can help you with these tasks by providing a visual interface for commiting individual changes in a file.

>  The change in date format needs to be done to ALL files, not just the template file. I'd rather not change it manually here.

Hm, with a proper rebase it should be no problem. Overall, I think the first PR maybe should be just the user manual, adding dates and makefile stuff later. If all is mixed together, this doesn't really help much.

Reply:
ok thanks, that's very helpful. I need to do a bit of restructuring work. 
Should I close all pull requests, then?

Reply:
No need to open new ones. When you have a new branch, you can force push it. I.e. if you want the first PR to be about the manual, you can push if `git push -f local_branch_name:github_branch_name`.
But you can also open new ones if want to restructure the PRs and think the current ones don't fit it.